pandas/core/generic.py

```diff
@@ -7366,16 +7366,32 @@ def resample(self, rule, how=None, axis=0, fill_method=None, closed=None,
                  label=None, convention='start', kind=None, loffset=None,
                  limit=None, base=0, on=None, level=None):
         """
+        Resample time-series data.
+
         Convenience method for frequency conversion and resampling of time
-        series.  Object must have a datetime-like index (DatetimeIndex,
-        PeriodIndex, or TimedeltaIndex), or pass datetime-like values
-        to the on or level keyword.
+        series.  Object must have a datetime-like index (``DatetimeIndex``,
+        ``PeriodIndex``, or ``TimedeltaIndex``), or pass datetime-like values
+        to the ``on`` or ``level`` keyword.
 
         Parameters
         ----------
         rule : string
-            the offset string or object representing target conversion
+            The offset string or object representing target conversion.
+        how : string
+            Method for down-/re-sampling, default to ‘mean’ for downsampling.
+
+            .. deprecated:: 0.18.0
+               The new syntax is ``.resample(...).mean()``, or
+               ``.resample(...).apply(<func>)``
         axis : int, optional, default 0
```

Comment:
Can you check other docstrings to see how the `axis` type is usually documented?

Reply:
Another example is:
```
apply to each column (``axis=0`` or ``'index'``)
or to each row (``axis=1`` or ``'columns'``) or    
```
The problem is that for `Series` the axis is that along which the operation is computed, which in this case it's always along the rows (aka *columnwise*).

Maybe a more verbose but informative doc would be 

```
 Which axis to use for up- or down-sampling. For ``Series`` this
 will default to 0, i.e. `along the rows`. Must be
 ``DatetimeIndex``, ``TimedeltaIndex`` or ``PeriodIndex``.
```

I have added this tentatively to the PR but let me know if something less verbose would be better.

Reply:
@jmrr can you make this change?

Reply:
Hi @datapythonista not your fault at all. I somewhat skipped the word _type_ in your request and I thought you meant how `axis` is usually documented. Regarding the type and values there's no much standardization either. From `set_axis()` in `pandas/core/generic.py` I like the following:

```{0 or 'index', 1 or 'columns'}, default 0```

If we extrapolate this to `Series` then that line should be:

```axis : {0 or 'index'}, default 0```

I will make this change now, hope it sounds good.